content/post/language_migrations.md

```diff
@@ -224,55 +189,25 @@ def power_iteration(A, nb_iterations=100, beta=0.85):
 
  power_iteration(transition_matrix)
 ```
-<p align="center" class="caption">Power Iteration algorithm, Python.</p>
+{{% center %}}
+Power Iteration algorithm, Python.
+{{% /center %}}
 
 
 ### Most popular languages on GitHub
 
 At last! Here is the reward: the stationary distribution of our [Markov chain](https://en.wikipedia.org/wiki/Markov_chain). This probability distribution is independent of the initial distribution. It gives information about the stability of the process of random switching between languages. Thus, no matter how popular the languages are at the present time, the hypothetical future stationary state stays the same. Here is the popularity ranking of our 25 languages used on GitHub:
 
-<style>
-table {
-    font-family: arial, sans-serif;
-    border-collapse: collapse;
-}
-td, th {
-    text-align: left;
-    padding: 8px;
-    word-wrap: break-word;
-}
-th {
-    background: #eee;
-}
-tr:nth-child(even) {
-    background-color: #dddddd;
-}
-.wrap {
-    margin-top:-30px;
-}
-.wrap table {
-    table-layout: fixed;
-}
-.inner_table {
-    margin-top:-48px;
-    height: 400px;
-    overflow-y: auto;
-    margin-bottom:30px;
-}
-</style>
-
-<div class="wrap">
-    <table class="head">
-      <table>
+<table>
+<thead>
     <tr>
       <th>Rank</th>
       <th>Language</th>
       <th>Popularity, %</th>
       <th>Source code, %</th>
     </tr>
-    </table>
-    <div class="inner_table">
-    <table>
+    </thead>
```

Comment:
Cause I'm not gonna rewrite to huge tables into pain-in-the-ass'y markdown table syntax by hand :stuck_out_tongue: don't think table markup syntax is gonna change any time soon so we should be fine